### ISSUE TITLE
Accent tags are inflated and unreadable.

### ISSUE THREAD

**Issue by goolies** [Aurorastation/Aurora.3#12128]
Instead of being able to read what accent a character has and the OOC and LOOC tags showing to the left when someone is using OOC I have what seems to be a very, very zoomed in tag instead. I've tried clearing my cache and messing with the settings inside Byond but nothing is working so far. Here's a screenshot of what they all look like.

![8824130078ed629be262f6f3c66a3dc4](https://user-images.githubusercontent.com/70824677/124186673-91cc8400-dab4-11eb-9756-cf58db5c942f.png)

**Comment by goolies** [comment]
I can only replicate it when I use windows DPI scaling which I use to make the text bigger because I have a 3840 x 2160 monitor (here's two differences side by side.) The first picture is the one with the option turned on in windows and the second is with it off.

![unknown (1)](https://user-images.githubusercontent.com/70824677/124189396-8d09cf00-dab8-11eb-8012-a0cd240a6887.png)

![unknown](https://user-images.githubusercontent.com/70824677/124189468-ab6fca80-dab8-11eb-94ab-4c61535f4b05.png)

![unknown (2)](https://user-images.githubusercontent.com/70824677/124189503-ba567d00-dab8-11eb-8808-02a1121617af.png)

**Comment by goolies** [comment]
I found a workaround for anyone who's on a higher resolution monitor. If you right click on the .exe of dreamseeker and go into properties, compatibility, change high DPI settings and apply the settings I've linked here it stops the tags from being zoomed in. You really have to mess around with the scaling of your computer's images and the percentage of how much it's scaling it.

I have my Windows scaling set to 200% percent but it could be different for every resolution. One of my friends has the same issue and when they set theirs to 150% scaling it fixed. Definitely turn on high DPI settings though if you're getting this.
![Capture](https://user-images.githubusercontent.com/70824677/124202037-72901f80-dad1-11eb-80b1-ee3a7fa3a66d.JPG)
![Capture2](https://user-images.githubusercontent.com/70824677/124202070-82a7ff00-dad1-11eb-9b52-9b22d73f427f.JPG)
